website/pages/en/deploying/hosted-service.mdx

```diff
@@ -2,7 +2,7 @@
 title: What is the Hosted Service?
 ---
 
-> Please note, the hosted service will begin sunsetting in 2023, but it will remain available to networks that are not supported on the decentralized network. Developers are encouraged to [upgrade their subgraphs to The Graph Network](/cookbook/upgrading-a-subgraph) as more networks are supported. Each network will have their hosted service equivalents gradually sunset to ensure developers have enough time to upgrade subgraphs to the decentralized network. Read more about the sunsetting of the hosted service [here](https://thegraph.com/blog/sunsetting-hosted-service).
+> Please note, hosted service endpoints will expire on END-DATE-PLACEHOLDER as all subgraphs will need to upgrade to The Graph Network. Please read more in the [Sunrise FAQ](../sunrise)
```

Comment:
I'd rather not have URLs relative to the current page, because they would break if the current page is moved (or this text is moved or copied to another page).

```suggestion
> Please note, hosted service endpoints will expire on END-DATE-PLACEHOLDER as all subgraphs will need to upgrade to The Graph Network. Please read more in the [Sunrise FAQ](/sunrise)
```